controller/metrics.go

```diff
@@ -103,6 +118,24 @@ var (
 		Name:      "net_rec",
 		Help:      "Usage memory",
 	}, []string{"vm_name", "host_name"})
+	prometheusVmPowerState = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Namespace: namespace,
+		Subsystem: "vm",
+		Name:      "power_state",
+		Help:      "poweredOn 1, poweredOff 2, standBy 3, other 0",
+	}, []string{"vm_name", "host_name"})
+	prometheusVmSwapUsage = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Namespace: namespace,
+		Subsystem: "vm",
+		Name:      "mem_swapped_average",
+		Help:      "Average VM swap usage",
+	}, []string{"vm_name", "host_name"})
+	prometheusVmMaxDiskLatency = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Namespace: namespace,
+		Subsystem: "vm",
+		Name:      "disk_maxTotalLatency_latest",
```

Comment:
I know old prometheus would barf on things other than snake case, but they might have been working on changing that. - Are we sure this is acceptable? Are there other instances in existing code like this?

Reply:
Yeah, I can't say I love this either.  It works and the reasoning being that our existing dashboard has the metric listed like this, but given that I'm already having to alter the dashboard, I'm going to change it.